src/cli/main.ts

```diff
@@ -0,0 +1,44 @@
+import yargs from 'yargs/yargs';
```

Comment:
Wouldn't this file rather be called `index.ts`?

Reply:
Done.